lib/internal/Magento/Framework/Stdlib/StringUtils.php

```diff
@@ -149,7 +149,7 @@ public function split($value, $length = 1, $keepWords = false, $trim = false, $w
      */
     public function strlen($string)
     {
-        return mb_strlen($string, self::ICONV_CHARSET);
+        return $string !== null ? mb_strlen($string, self::ICONV_CHARSET) : '';
```

Comment:
Hello @karyna-tsymbal-atwix.
I think this is a typo.
Please, fix it

```suggestion
        return $string !== null ? mb_strlen($string, self::ICONV_CHARSET) : 0;
```